data/tutorials/getting-started/gs_07_operators.md

```diff
@@ -0,0 +1,196 @@
+---
+id: operators
+title: Operators
+descrption: |
+  
+category: "Getting Started"
+---
+
+# Operators
+
+## Using Binary Operators
+
+In OCaml almost all binary operators are regular functions. The function underlying behind an operator is referred by surrounding the operator symbol with parenthesis. Here are the addition, string concatenation and equality functions:
```

Comment:
```suggestion
In OCaml, almost all binary operators are regular functions. The function underlying behind an operator is referred by surrounding the operator symbol with parentheses. Here are the addition, string concatenation, and equality functions:
```

I'm not clear what you mean by "underlying behind." It's usually one or the other. Is the function behind and operator or underlying it... or maybe even beneath it? Once I understand the meaning, I can revise the sentence.